File: python_ruby/oop_variable/2.py
class C:
    def __init__(self, v):
        self.value = v

    def show(self):
        print(self.value);
    def setValue(self, v):
        self.value = v
    def getValue(self):
        return self.value


c = C(10)
print(c.getValue())
c.setValue(2)
print(c.getValue())
c.show()

File: python_ruby/oop_variable/2.rb
class C
  def initialize(v)
    @value = v
  end

  def show()
    p @value
  end
  def setValue(v)
    @value = v
  end
  def getValue()
    return @value
  end
end
c = C.new(10)
p c.getValue
c.setValue(20)
p c.getValue()
c.show();
